src/basic/missing_sched.h

```diff
@@ -13,6 +13,13 @@
 assert_cc(CLONE_NEWCGROUP == 0x02000000);
 #endif
 
+/* b3e5838252665ee4cfa76b82bdf1198dca81e5be (5.2) */
+#ifndef CLONE_PIDFD
+#  define CLONE_PIDFD 0x00000080
```

Comment:
Hmm? This seems spurious though? Copy-paste error?

Reply:
Indeed. I somehow only changed the else part... Thanks for catching it. Fixed.